src/evaluations/data/evaluation_configs.py

```diff
@@ -96,15 +96,70 @@ def _smoke_test(num_runs=NUM_RUNS_VALUE):
       )
 
 
-def _generate_configs_scenario_3b(universe_size, num_sets, small_set_size,
+def _generate_configs_scenario_1_2(universe_size, num_sets, small_set_size,
+                                  large_set_size, remarketing_rate=None):
+  """Generate configs of Scenario 1 & 2
+  In this scenario,  publishers have heterogeneous users reach probability.
+  The reach probability of a user in a publisher is the same as that in other
+  publishers.
+
+  If remarketing_rate is not provided, we take it as scenario 1 with universe_size
+  as total. Othersize, we take it as scenario 2 with remarketing size as
+  int(universe_size*remarketing_rate)
+
+  See scenario 1 / 2:
+  Independent m-publishers / n-publishers independently serve a remarketing list
+  https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/doc/cardinality_and_frequency_estimation_evaluation_framework.md#scenario-1-independence-m-publishers-with-homogeneous-user-reach-probability
+  Returns:
+    A list of ScenarioConfigs of scenario 1 / 2 with selected parameters.
+  """
+  name_to_choices_of_set_size_list = {
```

Comment:
Seeing this set size setup in different scenarios, I'm thinking maybe better to merge them together in a single method for all scenarios 1,2,3,5? let me know if that makes sense.

Reply:
Makes sense to me.

Reply:
+1

Reply:
Added, I also updated scenario 1 to include `gradually_smaller`. There are more scenarios in 5 for different combinations of large/small set, but I didn't add it to default considering 5 is more sensitive to order and those are probably specifically designed for 5, but please let me know if it makes sense. Thanks!